core/src/main/java/com/graphhopper/routing/InstructionsFromEdges.java

```diff
@@ -391,12 +391,12 @@ private int getTurn(EdgeIteratorState edge, int baseNode, int prevNode, int adjN
             forceInstruction = true;
         }
 
-        InstructionsOutgoingEdges outgoingEdges = new InstructionsOutgoingEdges(prevEdge, edge, encoder, maxSpeedEnc, crossingExplorer, nodeAccess, prevNode, baseNode, adjNode);
-        int nrOfPossibleTurns = outgoingEdges.nrOfAllowedOutgoingEdges();
+        InstructionsOutgoingEdges outgoingEdges = new InstructionsOutgoingEdges(prevEdge, edge, encoder, maxSpeedEnc, roadClassEnc, roadClassLinkEnc, crossingExplorer, nodeAccess, prevNode, baseNode, adjNode);
+        int nrOfPossibleTurns = outgoingEdges.nrOfAllowedTurns();
```

Comment:
Can you refactor this to the normal Java style `getAllowedTurns`? (e.g. see GraphHopper.getWorkerThreads)

Reply:
Sure, I just pushed this change.